src/containers/UserSettings/settings.tsx

```diff
@@ -128,6 +128,11 @@ export const enableQueryStreamingOldBackendSetting: SettingProps = {
     description: i18n('settings.editor.queryStreaming.description'),
 };
 
+export const enableBlobStorageCapacityMetricsSetting: SettingProps = {
```

Comment:
is it possible to check if API is available for this feature before add a setting? (`useGetFeatureVersion`)